CMakeLists.txt

```diff
@@ -302,6 +308,15 @@ else()
 	endif()
 endif()
 
+if (USE_GNUTLS OR USE_MBEDTLS)
+	if (NOT NO_WEBSOCKET)
+		# Needed for SHA1, it should be present as GnuTLS/MbedTLS cryptography backend
```

Comment:
Unless I'm mistaken, MbedTLS does not rely on Nettle, so Nettle should not be linked against in that case. Instead, it looks like the SHA1 API is in `mbedtls/sha1.h`.

Reply:
:+1: Fixed, removed the nettle usage for mbed tls